src/main/java/org/isf/visits/gui/VisitView.java

```diff
@@ -779,10 +838,10 @@ private JPanel getWardPanel() {
 			}
 
 			wardBox.addActionListener(actionEvent -> {
-				Object selectedWard = wardBox.getSelectedItem();
+				ward = (Ward) wardBox.getSelectedItem();
 
-				if (selectedWard instanceof Ward) {
-					loadDataForWard((Ward) selectedWard);
+				if (ward instanceof Ward) {
```

Comment:
Given that line 841 casts the value to `Ward` isn't this statement always true?

Reply:
indeed it was necessary the `instanceof`, rolling back the change. Thanks!